tests/test_decl.py

```diff
@@ -451,6 +451,99 @@ def test_delete_operations_delete_attributes_if_no_list_of_uuids_was_given(
         assert len(root_function.functions) == 0
 
 
+class TestApplyExtend:
+    @staticmethod
+    def test_extend_operations_change_model_objects_in_place(
+        model: capellambse.MelodyModel,
+    ) -> None:
+        root_component = model.by_uuid(ROOT_COMPONENT)
+        fnc = model.by_uuid(ROOT_FUNCTION)
+        in_functions_before_apply = fnc in root_component.functions
```

Comment:
This should be directly `assert`ed here, because if this precondition isn't met, it doesn't make sense to even begin with the test. Fail fast. (Consider "checking whether the test environment makes sense" to be part of "preparing the test environment".)

If you want to make it *really* explicit, add a message to the assertion stating as much, like:

```python
assert fnc not in root_component.functions, "Precondition failed"
```